### ISSUE TITLE
Error when building site: toCSS (non-extended version)

### ISSUE THREAD

**Issue by ZZY2357** [wayjam/hugo-theme-mixedpaper#2]
Log:
```
Error: Error building site: TOCSS: failed to transform "style.css" (text/css): resource "css/style.css_48b060fe05b0a273d182ef83c0605941" not found in file cache
```

**Comment by wayjam** [comment]
Cannot reproduce at latest commit.

**Comment by jms301** [comment]
I'm getting the same error.

hugo version gives: 
Hugo Static Site Generator v0.72.0-8A7EF3CF linux/amd64 BuildDate: 2020-05-31T12:07:45Z

**Comment by jfmatth** [comment]
First, thanks for this Theme, it's really nice, just what I needed, but I'm running into the same error and did some troubleshooting and I found the cause of the error, but not sure why?

I'm running Hugo v.74.1

If I build a new site ``` hugo new site testing```, copy the examplesite files in, it works fine.  If I remove two files from the resources/_gen/assetts/css/style.css_* the error occurs.

If I build a new site, add some content, it works fine.

If I build a new site, add some content and then copy the examplesite config.toml file, it fails.

Something seems to be linking to those two files somehow

```
Error: Error building site: TOCSS: failed to transform "style.css" (text/css): resource "css/style.css_48b060fe05b0a273d182ef83c0605941" not found in file cache
```

Ill keep digging, but hope this helps you.  I

**Comment by wayjam** [comment]
@jms301 @jfmatth Thanks for details. I will check it.

**Comment by wayjam** [comment]
1. You **don't need to** delete the css assets. If theme has changed, run `hugo`, and it will generate new static site(including compile sass to css).
2. If you are using **Windows**, you need the “extended” Sass/SCSS Hugo version to compile the sass/scss. Follow the guidance here: <https://gohugo.io/getting-started/installing/>.

### How to use:
1. New a Hugo Site, `hugo new site mysite`
2. Add a theme, `git submodule add https://github.com/wayjam/hugo-theme-mixedpaper.git themes/mixedpaper`
3. Copy the `config.toml` from `themes/mixedpaper/exampleSite` to root directory of your site, and edit it.
4. Add some content.
5. Run `hugo` to generate or `hugo server` to preview.

@ZZY2357 @jms301 @jfmatth

**Comment by jfmatth** [comment]
@wayjam I followed your instructions above, and it **did not work :(**

I did not change the config.toml, but ran hugo serve -v --bind ... --baseURL ...


```
ubuntu@primary:~/development/hugo$ hugo new site mysite

Congratulations! Your new Hugo site is created in /home/ubuntu/development/hugo/mysite.

Just a few more steps and you're ready to go:

1. Download a theme into the same-named folder.
   Choose a theme from https://themes.gohugo.io/ or
   create your own with the "hugo new theme <THEMENAME>" command.
2. Perhaps you want to add some content. You can add single files
   with "hugo new <SECTIONNAME>/<FILENAME>.<FORMAT>".
3. Start the built-in live server via "hugo server".

Visit https://gohugo.io/ for quickstart guide and full documentation.
ubuntu@primary:~/development/hugo$ cd mysite/
ubuntu@primary:~/development/hugo/mysite$ git init
Initialized empty Git repository in /home/ubuntu/development/hugo/mysite/.git/
ubuntu@primary:~/development/hugo/mysite$ git submodule add https://github.com/wayjam/hugo-theme-mixedpaper.git themes/mixedpaper
Cloning into '/home/ubuntu/development/hugo/mysite/themes/mixedpaper'...
remote: Enumerating objects: 271, done.
remote: Counting objects: 100% (271/271), done.
remote: Compressing objects: 100% (178/178), done.
remote: Total 271 (delta 125), reused 209 (delta 67), pack-reused 0
Receiving objects: 100% (271/271), 233.62 KiB | 176.00 KiB/s, done.
Resolving deltas: 100% (125/125), done.

ubuntu@primary:~/development/hugo/mysite$ cp themes/mixedpaper/exampleSite/config.toml ./

ubuntu@primary:~/development/hugo/mysite$ hugo new posts/first.md
/home/ubuntu/development/hugo/mysite/content/posts/first.md created
ubuntu@primary:~/development/hugo/mysite$ hugo serve -v --bind 0.0.0.0  --baseURL primary.mshome.net
INFO 2020/07/15 11:23:07 No translation bundle found for default language "en"
INFO 2020/07/15 11:23:07 Translation func for language en not found, use default.
INFO 2020/07/15 11:23:07 i18n not initialized; if you need string translations, check that you have a bundle in /i18n that matches the site language or the default language.
INFO 2020/07/15 11:23:07 Using config file:
Building sites … INFO 2020/07/15 11:23:07 syncing static files to /
Built in 23 ms
Error: Error building site: TOCSS: failed to transform "style.css" (text/css): resource "css/style.css_48b060fe05b0a273d182ef83c0605941" not found in file cache
ubuntu@primary:~/development/hugo/mysite$
```

**Comment by wayjam** [comment]
@jfmatth what your version of ubuntu is?

> What this installs depends on your Debian/Ubuntu version. On Ubuntu bionic (18.04), this installs the non-extended version without Sass/SCSS support. On Ubuntu disco (19.04), this installs the extended version with Sass/SCSS support.
> This option is not recommended because the Hugo in Linux package managers for Debian and Ubuntu is usually a few versions behind as described [here](https://github.com/gcushen/hugo-academic/issues/703)

<https://gohugo.io/getting-started/installing/#debian-and-ubuntu>

**Comment by jfmatth** [comment]
@wayjam thanks for that update, seems to have fixed it, but weird, I don't know why?

After uninstalling hugo from a bin file locally, and clearing out libsass1 and reinstalling everthing, it's working fine, even without libsass1.

I typically don't use the ubuntu apt version since it's so old, and install the bin directly, however, for Ubuntu seems like it needs that library.

I think i'll do more testing on a fresh Ubuntu install to make sure I understand the issue.

thanks for the help.

**Comment by jfmatth** [comment]
Interesting, I did some testing and with 18.04 and it's a nightmare, not sure how I got mine to work.

I'm not satisfied that the Ubuntu experience is good, I've tried a combination of apt and snap, and none of them provide what I think is any good.  Looks like the apt install links to the library it needs, but snap doesn't install what it needs.  I wonder if a static link could help for HUGO, but not sure.

I did find a solution for Ubuntu 20.04 :

1. Install HUGO via ```apt-get install hugo```, which will install the necessary CSS library compiler
2. Download the latest .DEB package from Hugo releases page
3. install the .DEB package with ```dpkg -i hugo...```

I did the above steps with install .68.3-1 via apt, .74.1 via dpkg, and everything with this theme worked fine.
